.github/workflows/ci-actions.yml

```diff
@@ -486,7 +488,7 @@ jobs:
         if: matrix.mssql
       - name: Amazon Services
         run: |
-          docker run --rm --publish 8000:4569 --publish 8008:4572 --publish 8009:4575 --publish 8010:4576 --name build-amazon-service-clients -e SERVICES=s3,dynamodb,sns,sqs -e START_WEB=0 \
+          docker run --rm --publish 8000:4569 --publish 8008:4572 --publish 8009:4575 --publish 8010:4576 --publish 8011:4599 --publish 8012:4566 --name build-amazon-service-clients -e SERVICES=s3,dynamodb,sns,sqs,kms,ses -e START_WEB=0 \
```

Comment:
You probably need to make the timeout for the Amazon services native job higher. It's 20 right now, maybe make it 30.

Reply:
Yep, i noticed that